bin/qunit

```diff
@@ -0,0 +1,768 @@
+
+  def show_help
+    puts <<~HELP
+      Usage: bin/qunit [options] [files|directories]
+
+      Runs the Discourse QUnit suite through Ember Exam. With no arguments it runs
+      the entire suite against your dev servers; add --full to spin up an isolated
+      Rails/Ember stack (the same flow CI uses).
+
+      Options
+        -f, --filter TEXT          Filter tests by QUnit module/test name
+            --dry-run              Print what would run without executing
+            --ember-port PORT      Ember dev server port (default: 4200)
+            --rails-port PORT      Rails dev server port (default: 3000)
+            --full                 Manage the Rails/Ember servers for you
+            --parallel N           Number of workers when sharding tests (--full)
+            --seed SEED            Seed for deterministic ordering
+            --module TEXT          Restrict to a QUnit module
+            --target NAME          Force a core/plugins/<plugin> target
+            --qunit-skip-core      Skip core tests when targeting plugins/themes
+            --qunit-single-plugin NAME  Limit to one plugin payload
+            --qunit-path PATH      Serve tests from a custom path (themes)
+            --theme-name NAME      Theme metadata for theme runs
+            --theme-url URL
+            --theme-id ID
+            --rails-env ENV        Rails env for --full (default: test)
+            --report-requests      Print every HTTP request the suite makes
+            --load-plugins         Force plugin JS to load (auto-detected)
+        -v, --verbose             Verbose logging
+        -h, --help                Show this help
+
+      Notes
+        • Files or directories ending in *-test.{js,gjs} are passed straight to
+          Ember Exam via --file-path.
+        • Paths containing plugins/ automatically enable plugin assets.
+        • Environment variables (TARGET, MODULE, LOAD_PLUGINS, etc.) serve as
+          defaults for the corresponding flags.
+
+      Examples
+        bin/qunit frontend/discourse/tests/unit/models/user-test.js
+        bin/qunit --full plugins/chat/test/javascripts/unit/lib/chat-audio-test.js
+        bin/qunit --full --filter "Acceptance: Emoji"
+    HELP
+  end
+
+  def parse_options(args)
+    parser =
+      OptionParser.new do |opts|
+        opts.banner = "Usage: bin/qunit [options] [files/directories...]"
+
+        opts.on("-f", "--filter FILTER", "Run tests matching this filter") do |filter|
+          @filter = filter
+        end
+
+        opts.on("--dry-run", "Show what would run without executing") { @dry_run = true }
+
+        opts.on("--ember-port PORT", Integer, "Ember server port (default: 4200)") do |port|
+          @ember_port = port
+        end
+
+        opts.on("--rails-port PORT", Integer, "Rails server port (default: 3000)") do |port|
+          @rails_port = port
+        end
+
+        opts.on("--full", "--standalone", "Use full standalone mode (spins up own test server)") do
```

Comment:
`--full` doesn't feel intuitive to me. Originally I thought it meant "run the full test suite".

`--standalone` is good. Could we just go with that?